app/src/main/res/values/strings.xml

```diff
@@ -42,6 +45,10 @@
     <string name="docked_enabled">The system will emulate being in docked mode</string>
     <string name="username">Username</string>
     <string name="username_default">@string/app_name</string>
+    <string name="prod_keys">Prod keys location</string>
+    <string name="title_keys">Title keys location</string>
```

Comment:
We tend to use `Title Case` for settings also location is implied to some extent so isn't required, make this `Production Keys` and `Title Keys`